source/services/reddit.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Reddit service provider
+ *
+ * Notes:
+ * 1) How are shares counted?
```

Comment:
"How shares count" (not a question, just a title)

Reply:
Fixed